sample.env

```diff
@@ -20,6 +20,7 @@ AGENCY_PROVIDER_NUMBER=XXXXXXXXX
 DATABASE_ADAPTER=postgresql
 DATABASE_APP_DB=development_openpath_app
 DATABASE_CAS_DB=development_openpath_cas
+CAS_DATABASE_DB_TEST=test_openpath_cas
```

Comment:
Unrelated missing environment variable that caused `db:reset` to fail because the database didn't have a name.